invoiceManagement_backend/bin/db/changelog/changes/initial_ddl.sql

```diff
@@ -32,5 +32,17 @@ CREATE TABLE IF NOT EXISTS Worker
     ason date,
```

Comment:
1. asOn
2. rate- Include both  strate and otrate
2. Timesheet- Why strate and otrate is present in Timesheet table? Both of these should be part of worker table.